.github/CODEOWNERS

```diff
@@ -12,9 +12,9 @@
 /packages/analytics               @literat @crishpeen @curdaj @pavelklibani @lmc-eu/spirit-design-system
 /packages/codemods                @literat @adamkudrna @crishpeen @pavelklibani @curdaj @lmc-eu/spirit-design-system
 /packages/common                  @literat @pavelklibani @curdaj @lmc-eu/spirit-design-system
-/packages/design-tokens           @literat @adamkudrna @crishpeen @curdaj @pavelklibani @katerinarehorkova @lmc-eu/spirit-design-system
+/packages/design-tokens           @literat @adamkudrna @crishpeen @curdaj @pavelklibani @katerinarehorkova @vmilersky @dlouhak @lmc-eu/spirit-design-system
 /packages/form-validations        @literat @pavelklibani @curdaj @lmc-eu/spirit-design-system
 /packages/icons                   @literat @adamkudrna @crishpeen @lmc-eu/spirit-design-system
```

Comment:
Could you please add me as owner of this package as well? Thank you. :)

Reply:
Added.